src/asl.ts

```diff
@@ -513,6 +518,15 @@ const FUNCTIONLESS_CONTEXT_NAME = "fnl_context";
  */
 const FUNCTIONLESS_CONTEXT_JSON_PATH = `$.${FUNCTIONLESS_CONTEXT_NAME}`;
 
+export interface ASLOptions {
+  /**
+   * Options for the ASL Optimizer.
+   *
+   * Default: Suggested
```

Comment:
What does this mean

Reply:
The default is the suggested set of optimizations, haha. I can elaborate.

Reply:
or this

```ts
   * @default ASLOptimizer.DefaultOptimizeOptions
```

Reply:
Use @link but yes